### DISCUSSION TITLE
Visual means of "content parity" on the translated page

### DISCUSSION THREAD

**Discussion by hochan222** [mdn/mdn-community#333] [Localization]
## Issue background

We got a way to record the "content parity" of localized documents in [How to indicate "content parity" of localized documents vs en-US counterparts?](https://github.com/mdn/mdn-community/discussions/129).

## Suggestion

At this point, I'm of the opinion that it would be nice if there was a visual means of representing that information on the translation page.

Even if it's not detailed, the translated page can gain a lot of trust from people who read MDN just by showing approximate status like red, green, and orange traffic lights. Alternatively, we can provide a funnel for page contributions to potential contributors.

First of all, I'd like to ask for your opinion on the comment. If you agree, I would like to specify the criteria for how and where on the page below in this discussion. 

## Solution

How about include that information in the blue notification box.

- blue: less than 10 commits behind
- orange: less than 30 commits behind
- red: Pages with hash information right now..?
- grey: no information

<img width="1699" alt="image" src="https://user-images.githubusercontent.com/22424891/219846423-e6fa86c2-9e1c-4fd4-8cb7-f2ded9683cce.png">

**Comment by AnuchaRodpha** [comment]
no information
